src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -435,6 +435,71 @@ private void Goo(List<string?> l)
 }");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        public async Task TestInvocationWithClassConstrainedTypeArgumentAndNullableTypes()
+        {
+            await TestInRegularAndScriptAsync(
```

Comment:
Can we make sure to test nullable constraint with not-null flow analysis? 

```c#
class C 
{
    public T M<T>() where T : class?, new
    {
        var t = new T();
        return [|M2|](t);
    }
}
```

Reply:
Isn't that the TestInvocationWithNullableClassConstrainedTypeArgumentAndNullableTypes test below?

Reply:
oof, mondays...